homeassistant/components/bluetooth_le_tracker/device_tracker.py

```diff
@@ -26,6 +27,13 @@ def setup_scanner(hass, config, see, discovery_info=None):
     # pylint: disable=import-error
     import pygatt
     new_devices = {}
+    adapter = None
+
+    async def async_stop():
```

Comment:
Event listener callbacks need to take `event` as argument or else you get an exception when you close, and nothing is killed.